langchain/src/chains/question_answering/load.ts

```diff
@@ -1,91 +1,48 @@
 import { LLMChain } from "../llm_chain.js";
-import { BasePromptTemplate } from "../../prompts/index.js";
-import {
-  StuffDocumentsChain,
-  MapReduceDocumentsChain,
-} from "../combine_docs_chain.js";
+import { StuffDocumentsChain, MapReduceDocumentsChain, } from "../combine_docs_chain.js";
 import { QA_PROMPT_SELECTOR, DEFAULT_QA_PROMPT } from "./stuff_prompts.js";
-import {
-  COMBINE_PROMPT,
-  DEFAULT_COMBINE_QA_PROMPT,
-  COMBINE_PROMPT_SELECTOR,
-  COMBINE_QA_PROMPT_SELECTOR,
-} from "./map_reduce_prompts.js";
-import { BaseLanguageModel } from "../../base_language/index.js";
-
-interface qaChainParams {
-  prompt?: BasePromptTemplate;
-  combineMapPrompt?: BasePromptTemplate;
-  combinePrompt?: BasePromptTemplate;
-  type?: string;
-}
-export const loadQAChain = (
-  llm: BaseLanguageModel,
-  params: qaChainParams = {}
-) => {
-  const {
-    prompt = DEFAULT_QA_PROMPT,
-    combineMapPrompt = DEFAULT_COMBINE_QA_PROMPT,
-    combinePrompt = COMBINE_PROMPT,
-    type = "stuff",
-  } = params;
-  if (type === "stuff") {
-    const llmChain = new LLMChain({ prompt, llm });
+import { COMBINE_PROMPT, DEFAULT_COMBINE_QA_PROMPT, COMBINE_PROMPT_SELECTOR, COMBINE_QA_PROMPT_SELECTOR, } from "./map_reduce_prompts.js";
```

Comment:
Looks like you're changing a lot of unrelated lines? If you could clean up the diff then we can review your changes